extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -190,8 +191,9 @@ export class AmpA4A extends AMP.BaseElement {
 
   /**
    * @param {!Element} element
+   * @param {number} upgradeDelayMs
    */
-  constructor(element) {
+  constructor(element, upgradeDelayMs) {
```

Comment:
I've added @lannka for review on our side. Generally, I think it's ok. I could also suggest adding `upgradeStartTime` property in the `CustomElement` and use it to calc deltas - this way you can avoid changing all the constructors.